package.yaml

```diff
@@ -43,10 +43,14 @@ default-extensions:
   - UnicodeSyntax
 
 when:
-  - condition: impl(ghc < 8.6)
+  condition: impl(ghc < 8.6)
+  then:
     default-extensions:
       - MonadFailDesugaring
       - TypeInType
+  else:
```

Comment:
Let's write this as a separate condition, since it's not actually related to the `else` branch.